In [6]:
import numpy as np
import sys

In [7]:
a = list(range(10000))

In [8]:
sys.getsizeof(a)

80056

In [13]:
b = np.array(a)

In [15]:
sys.getsizeof(b)

80104

The size of the list is 80056, however the side of a numpy array is 
80104